features/autorisatie/bewoning/raadpleeg-bewoning-op-peildatum/autorisatie.feature

```diff
@@ -0,0 +1,234 @@
+# language: nl
+
+@autorisatie
+Functionaliteit: autorisatie voor het gebruik van de BRP API Bewoning met peildatum
+  Autorisatie voor het gebruik van Bewoning gebeurt op twee niveau's:
+  1. autorisatie van de gebruiker door de afnemende organisatie
+  2. autorisatie van de afnemer (organisatie) door RvIG
+
+  Deze feature beschrijft alleen de autorisatie van de afnemer door RvIG.
+
+  De BRP API Bewoning wordt alleen aangeboden aan gemeenten voor het raadplegen de bewoning van adresseerbare object binnen de eigen gemeente.
+
+  Autorisatie wordt verkregen met behulp van een OAuth 2 token. 
+  Wanneer de afnemer een gemeente is, bevat het OAuth token een gemeentecode claim. Deze wordt gebruikt om te bepalen of het bevraagde adres binnen de eigen gemeente ligt.
+  Autorisatie voor bewoning wordt bepaald door de gemeente waar het gevraagde adresseerbaar object ligt.
+
```

Comment:
deze laatste zin kun je op 2 manieren lezen, nl ook: de gemeente bepaalt wie er wordt geautoriseerd :-).
Wat bedoeld wordt is: een gemeente is geautoriseerd voor Bewoning als het adresseerbaar object waarvan de Bewoning wordt gevraagd in de gemeente ligt, of ooit in de gemeente heeft gelegen.

Misschien van "door" "met behulp van" vervangen?